.circleci/config.yml

```diff
@@ -89,28 +97,38 @@ jobs:
 
   test_macos:
     macos:
-      xcode: "9.0"
+      xcode: "11.6.0"
     steps:
       - checkout
+      - run: HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv
```

Comment:
Tried the "xcode 12" [executor](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v3358/index.html) the contains system Python 3.8.5 (pip 20.1.1) but tests failed with SSL errors - it's not compiled against openssl libs correctly so have to go with a pyenv-installed Python

Running `HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv` takes 14 minutes :/ https://circleci.com/gh/artsy/hokusai/1156